docs/SIL.rst

```diff
@@ -437,7 +437,13 @@ number of ways:
     a guaranteed direct parameter.
 
   - Otherwise, the context value is treated as an unowned direct
-    parameter.
+    parameter. FIXME!!! what is this?? It sounds like ``@noescape
```

Comment:
This predate ``@noescape``. ``@convention(block)``'s context is passed unowned

From test/IRGen/objc_block.sil:

```
// CHECK-LABEL: define{{( protected)?}} swiftcc %T10objc_block3FooC* @call_block(%objc_block*, %T10objc_block3FooC*) {{.*}} {
// CHECK:       entry:
// CHECK:         [[T0:%.*]] = getelementptr inbounds %objc_block, %objc_block* %0, i32 0, i32 3
// CHECK:         [[T1:%.*]] = load i8*, i8** [[T0]]
// CHECK:         [[T2:%.*]] = bitcast i8* [[T1]] to i8* (%objc_block*, i8*)*
// CHECK:         [[T3:%.*]] = bitcast %T10objc_block3FooC* %1 to i8*
// CHECK:         [[T4:%.*]] = call i8* [[T2]](%objc_block* %0, i8* [[T3]])
// CHECK:         [[T5:%.*]] = bitcast i8* [[T4]] to %T10objc_block3FooC*
// CHECK:         ret %T10objc_block3FooC* [[T5]]
// CHECK:       }
```